domain/permission/state/state.go

```diff
@@ -57,49 +58,24 @@ func (s *State) CreatePermission(ctx context.Context, newPermissionUUID uuid.UUI
 		return userAccess, errors.Trace(err)
 	}
 
-	// Insert a permission doc with
-	// * permissionObjectAccess as permission_type_id
-	// * uuid of the user (spec.User) as grant_to
-	// * spec.Target.Key as grant_on
-	newPermission := `
-INSERT INTO permission (uuid, permission_type_id, grant_on, grant_to)
-VALUES ($M.uuid, $M.access_type_id, $M.grant_on, $M.grant_to)
-`
-	insertPermissionStmt, err := sqlair.Prepare(newPermission, sqlair.M{})
-	if err != nil {
-		return userAccess, errors.Trace(err)
-	}
-
-	m := make(sqlair.M, 2)
```

Comment:
This should have been `m := make(sqlair.M, 4)`. I moved it to a strong type, to improve readability.